<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecretConfigDialog</class>
 <widget class="QDialog" name="SecretConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secrets Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listSecret">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="editDescription">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="editValue">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemoveAll">
       <property name="text">
        <string>Remove All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonImport">
       <property name="text">
        <string>Import From...</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listSecret</tabstop>
  <tabstop>editDescription</tabstop>
  <tabstop>buttonRemove</tabstop>
  <tabstop>buttonRemoveAll</tabstop>
  <tabstop>buttonImport</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SecretConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SecretConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
